### PR TITLE
Removed dependency to logging library

### PR DESCRIPTION
**Pull request by favreau**

### CODE REVIEW

File: rendering_resource_manager_service/utils/custom_logging.py [hunk 1]
```diff
@@ -41,11 +60,7 @@ def info(level, message):
     :param level: Message level. Only message with a level higher than LOGGING_LEVEL are processed
     :param message: Message to be logged
     """
-    L.info("[%s] [%s] [%s] %s",
-           str(level),
-           datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S'),
-           current_thread().getName(),
-           message)
+    print_message('[INFO ]', sys.stdout, level, message)
```
Comment (rdumusc) [inline rendering_resource_manager_service/utils/custom_logging.py:63]: extra ' ' after INFO :-)

File: rendering_resource_manager_service/utils/custom_logging.py [hunk 2]
```diff
@@ -26,12 +26,31 @@
...
 
 from threading import current_thread
-
 import datetime
-import logging
+from threading import Lock
+import sys
+
+log_mutex = Lock()
+log_level = 1
```
Comment (rdumusc) [inline rendering_resource_manager_service/utils/custom_logging.py]: rename LOGGING_LEVEL as in the doc of functions below?

**Comment by rdumusc** [comment]
+1, looks good, just 2 minor details

File: rendering_resource_manager_service/utils/custom_logging.py (hunk 1 above)
Comment (favreau) [inline rendering_resource_manager_service/utils/custom_logging.py:63]: Well yes, it's to keep an alignment with DEBUG and ERROR. I think it makes the logs easier to read. What do you think?

Comment (rdumusc) [inline rendering_resource_manager_service/utils/custom_logging.py:63]: Ah OK, that's fine then!

**Pull request merged**